ghost.py

```diff
@@ -453,10 +523,11 @@ def _prettify_list(items):
     token_normalize_func=lambda param: param.lower())
 
 # Currently static. We'll see how backend implementations go and adjust
-# to dymanic mapping accordingly
+# to dynamic mapping accordingly
 STORAGE_NAME_MAPPING = {
     'tinydb': TinyDBStorage,
     'sqlalchemy': SQLAlchemyStorage,
+    'consul': ConsulStorage
```

Comment:
um.. this doesn't actually add support for consul in the CLI since it doesn't allow the user to pass the relevant arguments to the class other than the host and port. Are we ok with that?